pkg/registration/register.go

```diff
@@ -15,8 +15,8 @@ import (
 func RegisterUser(ctx context.Context, username string, password string, email string,
 	userStore user.Store, bot bool, argonConfig config.ArgonConfig) error {
 	// username = strings.Rep
-	if len(username) < 1 || len(username) > 20 {
-		return errors.New("username must be between 1 and 20 letters in length")
+	if len(username) < 3 || len(username) > 20 {
+		return errors.New("username must be between 3 and 20 letters in length")
```

Comment:
this would show only after submission, like this:

<img width="651" alt="Screenshot 2021-02-01 at 08 00 06" src="https://user-images.githubusercontent.com/4179698/106402245-90deea00-6463-11eb-8416-7128dbb4bade.png">

consider also adjusting [`register.tsx`](https://github.com/domino14/liwords/blob/a1bebf117333735e53d1abe9de15a2add8206d68/liwords-ui/src/lobby/register.tsx#L98-L117) to get as-they-type feedback:

<img width="315" alt="Screenshot 2021-02-01 at 07 47 29" src="https://user-images.githubusercontent.com/4179698/106402075-c3d4ae00-6462-11eb-96a2-92af65264147.png">

```diff
--- a/liwords-ui/src/lobby/register.tsx
+++ b/liwords-ui/src/lobby/register.tsx
@@ -103,6 +103,10 @@ export const Register = () => {
                   required: true,
                   message: 'Please input your username',
                 },
+                {
+                  min: 3,
+                  message: 'Min username length is 3',
+                },
                 {
                   max: 20,
                   message: 'Max username length is 20',
```

Reply:
Good catch. Added.